hack/generated/controllers/controller_test.go

```diff
@@ -53,7 +55,6 @@ func Test_ResourceGroup_CRUD(t *testing.T) {
 }
 
 func Test_StorageAccount_CRUD(t *testing.T) {
-	t.Parallel()
```

Comment:
Removing this intentional?

Reply:
Seems strange the above test has it but this doesn't

Reply:
Was removed while testing. Restored now.